### PR TITLE
Adds tiff files when using images.fromtif default 'tif extension

### PR DESCRIPTION
**Pull request by boazmohar**
Solves #341 
@freeman-lab What is a good default behavior as this is implemented now, it will not enable you to select either tif or tiff from your folders.
I could change the default `ext='tif'` to `'ext='tifs' to make it possible, What do you think?
Also didn't test with s3, any thoughts as to how?

### CODE REVIEW

**Comment by freeman-lab** [comment]
Cool @boazmohar ! I agree this would be good to support, another idea might be to just say if either `tif` or `tiff` is specified, we look for all of them? In other words, make it symmetric, whereas what you have only handles one of the cases. What do you think about that?

File: test/test_readers.py [hunk 1]
```diff
@@ -74,3 +74,24 @@ def test_local_recursive_nested(tmpdir):
     make(tmpdir, filenames)
     actual = LocalFileReader().list(str(tmpdir), recursive=True)
     assert parse(actual) == expected
+
```
Comment (freeman-lab) [inline test/test_readers.py]: These should probably be in `test_images_io.py`, the `test_readers.py` are for testing generic io functionality relevant to all data types

**Comment by freeman-lab** [comment]
For testing on `s3`, you could at least make sure `thunder.images.fromexample('fish')` works for you after this change, that loads `tif` files from `s3` with the default settings, so is at least a good sanity check for this fix. BTW we don't include that one in the test suits because it takes extra time and requires a network connection.

**Comment by boazmohar** [comment]
@freeman-lab Sure I can make it symmetric but what about a case where someone wants only tif or only tiff files. If you are OK without these options I am too. I thought to have a default of 'tifs' and that would be 'symmetric' as it will include tif and tiff. Specifying another extension will work as expected.
Tell me what you prefer, I don't have a string opinion either way.

File: test/test_readers.py (hunk 1 above)
Comment (boazmohar) [inline test/test_readers.py]: I tried moving them but I would need to either import the `make` and `parse` functions or copy paste them into `test_images_io.py`. I think they belong in the `test_readers.py` that is the main function that I changed.

**Comment by boazmohar** [comment]
@freeman-lab fish example is working. Thanks.

File: test/test_readers.py (hunk 1 above)
Comment (freeman-lab) [inline test/test_readers.py]: Ah I see, that's fine then.

**Comment by freeman-lab** [comment]
Ah, totally see where you were coming from, I think my call would be to just make it symmetric, as both extensions really do fundamentally specify the same format (see e.g. http://superuser.com/questions/287776/differences-between-tiff-tif-tiff-tif), and it seems very unusual for people to have both in the same location.

**Comment by boazmohar** [comment]
@freeman-lab Thanks for the feedback, ready to merge in my opinion.

**Comment by freeman-lab** [comment]
ok great @boazmohar , merging this in!

**Pull request merged**